example/zend-3.0.3/CONDUCT.md

```diff
@@ -0,0 +1,43 @@
+# Contributor Code of Conduct
```

Comment:
For a minimal example I would consider removing markdown files not related to Bugsnag to avoid confusion.